.github/workflows/aws.yml

```
@@ -104,12 +104,35 @@ jobs:
+             elif .name == "RDS_URL" then .value = $rds_url
+             elif .name == "RDS_USERNAME" then .value = $rds_username
+             elif .name == "RDS_PASSWORD" then .value = $rds_password
+             else .
+             end
+           )
+           ' .aws/obtoo-def-revision2.json > "$tmpfile" && mv "$tmpfile" .aws/obtoo-def-revision2.json
+
+    - name: Register ECS Task Definition
+      run: |
+        aws ecs register-task-definition --cli-input-json file://.aws/obtoo-def-revision2.json
+
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 128: did not find expected key

(syntax-check)